go.mod

```diff
@@ -0,0 +1,122 @@
+module github.com/pingcap/tidb
+
+require (
+	cloud.google.com/go v0.31.0 // indirect
+	github.com/BurntSushi/toml v0.3.1
+	github.com/Shopify/sarama v1.19.0 // indirect
+	github.com/Shopify/toxiproxy v2.1.3+incompatible // indirect
+	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7 // indirect
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
+	github.com/blacktear23/go-proxyprotocol v0.0.0-20171102103907-62e368e1c470
+	github.com/chzyer/readline v0.0.0-20171208011716-f6d7a1f6fbf3 // indirect
+	github.com/cockroachdb/cmux v0.0.0-20170110192607-30d10be49292 // indirect
+	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/coreos/bbolt v1.3.1-coreos.6 // indirect
+	github.com/coreos/etcd v3.3.10+incompatible
```

Comment:
Why there is an `incompatible`? Does it mean that we have dependencies that import different versions of etcd?